vkcs/networking/resource_port.go

```diff
@@ -402,6 +412,9 @@ func resourceNetworkingPortRead(ctx context.Context, d *schema.ResourceData, met
 	// This can be different from what the user specified since
 	// the port can have the "default" group automatically applied.
 	d.Set("all_security_group_ids", port.SecurityGroups)
+	if d.Get("full_security_groups_control").(bool) {
+		d.Set("security_group_ids", port.SecurityGroups)
+	}
```

Comment:
Could we write a deprecation message if full_security_groups_control has false value?

Reply:
add mesage